akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -740,6 +738,18 @@ private[akka] class Shard(
       }
 
     case RestartRememberedEntities(ids) => restartEntities(ids)
+    case ShardIdsMoved(entityIds) =>
```

Comment:
Is this not EntityIdsMoved? The shards haven't moved

Reply:
Ups, you are right